models/TMQ-WS850-VRT850-PSL-.001-wce/config.json

```diff
@@ -30,5 +30,6 @@
     ],
     "loss": "weighted_cross_entropy",
     "weights": [0.355, 72.171, 5.875],
+    "save_epochs": true,
```

Comment:
I don't think we need this in config, we can just comment out the line in example.py or delete the model